### ISSUE TITLE
hi, how can i join this group ???

### ISSUE THREAD

**Issue by glory2012** [deepin-community/sig-deepin-surface#1]

**Comment by zccrs** [comment]
Thank you. Are you a developer or a surface user?

**Comment by glory2012** [comment]
yes, i am a developer and i am seeking a solution for deepin support in arm64 platform

**Comment by zccrs** [comment]
> yes, i am a developer and i am seeking a solution for deepin support in arm64 platform

OK, Is Surface X the first target for arm64? For the Surface X, you can join https://matrix.to/#/#deepin-surface:mozilla.org , and I will add you to the https://github.com/deepin-community/SIG/blob/master/sig/deepin-surface/MEMBERS.md#%E5%B0%8F%E7%BB%84%E6%88%90%E5%91%98 members list.
Can you speak Chinese? Sorry for the documents of deepin community is not have English contents now.

**Comment by glory2012** [comment]
可以，能帮忙把我加一下上面的讨论组吗，非常感谢

**Comment by glory2012** [comment]
> 可以，能帮忙把我加一下上面的讨论组吗，非常感谢

**Comment by zccrs** [comment]
> > 可以，能帮忙把我加一下上面的讨论组吗，非常感谢

这就更好说了，交流也无障碍。上面发的那个是个 Matrix 讨论组，可以注册一个 mozilla 的 matrix 服务的账号，然后发给我名字，我邀请你进到这个组。另外我一会先把你的 github 账户加到 deepin-surface 的成员列表。

**Comment by zccrs** [comment]
已更新：https://github.com/deepin-community/SIG/blob/master/sig/deepin-surface/MEMBERS.md
欢迎你加入 deepin surface 适配组
